ui/src/components/Rules/Rule/RuleCreator/RuleInput/index.tsx

```diff
@@ -355,208 +349,223 @@ const RuleInput = (props: IRuleInput) => {
                     },
                   )}
                 </select>
+                <button
+                  className="ml-1 p-1 text-sm"
+                  onClick={(e) => {
+                    e.preventDefault()
+                    props.onOpenHelpModal()
+                  }}
```

Comment:
You don't need preventDefault here, set the type to button. By default it'll be type=submit when in a `<form>`

```suggestion
                  type="button"
                  onClick={props.onOpenHelpModal}
```

Reply:
No way that is the only thing you saw that needed an update/change.

Reply:
I haven't finished yet, I started and got tired 😆 . I'll have more of a look soon!

Reply:
I got rid of the help modal all together, including that button.